src/util/event_loop_dispatcher.hpp

```diff
@@ -54,11 +54,11 @@ class EventLoopDispatcher<void(Args...)> {
     EventLoopDispatcher(std::function<void(Args...)> func)
     : m_state(std::make_shared<State>(std::move(func)))
     {
-        m_scheduler->set_notify_callback([state = m_state] {
+        m_scheduler->set_notify_callback([state = m_state, this] {
```

Comment:
```suggestion
        m_scheduler->set_notify_callback([state = m_state] {
```

Reply:
good catch. It seems this file was not merged correctly.